stdlib/log-api/src/main/ballerina/src/log/natives.bal

```diff
@@ -16,6 +16,17 @@
 
 import ballerina/java;
 
+# Represents log level types.
+public enum LogLevel {
+    DEBUG = "DEBUG",
```

Comment:
You don't need to say `DEBUG = "DEBUG"`. Just `DEBUG` is enough. Only need to do so if you want to give a different string value to the enum value `DEBUG`

Reply:
Ack.